examples/cluster/bottlerocket_gpu_node_group/main.tf

```diff
@@ -24,6 +24,9 @@ module "node_group" {
   instance_types = ["g4dn.xlarge"]
   min_size       = 1
 
+  imdsv2_required = true
```

Comment:
do we need to set this everywhere in the examples - it is the default?

Reply:
the default is true, I'll remove it from here just to make it cleaner